### ISSUE TITLE
Automatic launch of test servers for PRs?

### ISSUE THREAD

**Issue by farooqkz** [MT-CTF/capturetheflag#1100]
I was thinking about it: Would a service which automatically launches a public test server for PRs be useful for testing projects like this?

**Comment by LoneWolfHT** [comment]
The issue is coordinating times when I can be there to see people testing. It's pretty easy to set up a test server

**Comment by farooqkz** [comment]
> The issue is coordinating times when I can be there to see people testing. It's pretty easy to set up a test server

Yeah but I was thinking about making this process automatic. Minetest-as-a-Service?

**Comment by FlamingRCCars** [comment]
This would be extremely simple with a batch script, the real question is should you? Testing is most effective when the developer is online, and having it up at all times could cause people to not join it, but on the other hand this would make testing much quicker
